servers/src/main/java/tachyon/worker/block/evictor/EvictorType.java

```diff
@@ -30,6 +30,13 @@
    */
   LRU,
   /**
+   * Evict old blocks in certain StorageDir by LRU. The main difference between PartialLRU and
+   * LRU is that LRU choose old blocks among several StorageDirs until one StorageDir
+   * satisfies the request space, but PartialLRU select one StorageDir first and evict old
+   * blocks in certain StorageDir by LRU
+   */
+  PARTIAL_LRU,
```

Comment:
@calvinjia I think this is logic of LRU Evictor. LRU Evictor simulate LRU eviction in each directory until a dir has enough space. I have referred the code of [branch-0.6](https://github.com/amplab/tachyon/blob/branch-0.6/core/src/main/java/tachyon/worker/eviction/EvictPartialLRU.java), I think the Partial LRU Evictor mainly focus on choosing dir with max free space to ensure the uniform distribution of blocks.